technical-advisory-groups/security-tag/README.md

```diff
@@ -0,0 +1,21 @@
+# Security TAG
+
+## Mission Statement
+The OpenSearch Security Technical Advisory Group (TAG) champions the strategic direction and technical guidance for security solutions within the OpenSearch ecosystem.
```

Comment:
It's plausible that people looking to report a security issue could find there way here. Might be worth linking to the [security response documentation](https://github.com/opensearch-project/.github/blob/main/SECURITY.md) somewhere near the top.